x-pack/plugins/lens/public/datatable_visualization/sorting.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import ipaddr from 'ipaddr.js';
+import type { IPv4, IPv6 } from 'ipaddr.js';
+import { FieldFormat } from 'src/plugins/data/public';
+
+function isIPv6Address(ip: IPv4 | IPv6): ip is IPv6 {
+  return ip.kind() === 'ipv6';
+}
+
+function getSafeIpAddress(ip: string) {
+  if (!ipaddr.isValid(ip)) {
+    // for non valid IPs have the same behaviour as for now (we assume it's only the "Other" string)
+    // create a mock object which has all -Infinity values to be the last item
+    return { parts: Array(8).fill(-Infinity) };
```

Comment:
I think the result of this is that Other is always shown at the beginning of the list. Can we push it to the end instead, since I would want to show IPs in ascending order?